keep/api/routes/alerts.py

```diff
@@ -617,6 +623,16 @@ async def receive_generic_event(
         alert,
         pusher_client,
     )
+
+    if authenticated_entity.api_key_name:
+        logger.info("Updating API Key last used")
```

Comment:
change to debug

Reply:
fixed by d3c5e5ed23035e56ae8a9354499d6521451917e6